use-cases/conference-scheduling/src/main/java/org/acme/conferencescheduling/domain/Talk.java

```diff
@@ -541,22 +535,6 @@ public void setCrowdControlRisk(int crowdControlRisk) {
         this.crowdControlRisk = crowdControlRisk;
     }
 
-    public Timeslot getPublishedTimeslot() {
-        return publishedTimeslot;
-    }
-
-    public void setPublishedTimeslot(Timeslot publishedTimeslot) {
-        this.publishedTimeslot = publishedTimeslot;
-    }
-
-    public Room getPublishedRoom() {
-        return publishedRoom;
-    }
-
-    public void setPublishedRoom(Room publishedRoom) {
-        this.publishedRoom = publishedRoom;
-    }
-
     public boolean isPinnedByUser() {
         return pinnedByUser;
     }
```

Comment:
This is also related to publishing, isn't it?
We don't have a feature in the UI to pin anything, so arguably the entire support for pinning can go too.

Reply:
I don't think it is related, but we are not using it. Let me remove it.